Tuto-GUDHI-perslay-visu.ipynb

```diff
@@ -0,0 +1,229 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    import gudhi.tensorflow as gdtf
</pre><p>Should be <code>import gudhi.tensorflow.perslay as gdtf</code> or <code>import gudhi.tensorflow.perslay as prsl</code> (but requires modifications in the notebook.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GUDHI/TDA-tutorial/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>